docs/standard/analyzers/portability-analyzer.md

```diff
@@ -1,41 +1,74 @@
 
-If you don’t want to use Visual Studio, you can use the Portability Analyzer from the command prompt. Just download the API Portability Analyzer from the [Microsoft/dotnet-apiport](https://github.com/Microsoft/dotnet-apiport/releases) repository.
+If you want to use [ApiPort console app](https://aka.ms/apiportdownload). 
 
 * Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
 * To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
+* Run `\...\ApiPort.exe -?` to get more help
+
+It is recommended that you include all the related exe and dll files that you own and want to port, and exclude the files that your app depends on, but you don't own and can't port. This will give you most relevant portability report.  
+
+### View and interpret portability result
+
+Only APIs that are unsupported by a Target Platform appear in the report. 
+After running the analysis in Visual Studio, you'll see your .NET Portability report file link pops up. If you used [ApiPort console app](https://aka.ms/apiportdownload), your .NET Portability report is saved as a file in the format you specified. The default is in an Excel file (*.xlsx*) in your current directory. 
+
+#### Portability Summary 
+
+![Portability Summary](./media/portability-analyzer/PortabilitySummary.PNG)
 
-Your .NET Portability Report is saved as an Excel file (*.xlsx*) in your current directory. The **Details** tab in the Excel Workbook contains more information.
+The Portability Summary section of the report shows the the portability percentage for each assembly included in the run. In the previous example, 89.74% of the .NET Framework APIs used in the `ConsoleAppFramework` app are available in .NET Core + Platform Extensions v2.2. If you run the .NET Portability Analyzer tool against multiple assemblies, each assembly should have a row in the Portability Summary report.
+
+#### Details
+
+![Portability Details](./media/portability-analyzer/PortabilityDetails.PNG)
+
+The Details section of the report lists the APIs missing from one of the Target Platforms. 
+
+ - Target type: the type has missing API from a Target Platform 
+ - Target member: the method is missing from a Target Platform 
+ - Assembly name: the .NET Framework assembly that the missing
+   API lives in. 
+ - Each of the selected Target Platforms is one column, such as ".NET Core": "Not supported" value means the API is not
+   supported on this Target Platform. 
+ - Recommended Changes: recommended API or technology to change to. Currently, this field is empty or out of date for a lot of APIs. Due to the large number of APIs, we have big challenge to keep it up. We are looking at alternate solutions to provide helpful information to customers.
+
+#### Missing Assemblies
+
+![Portability Details](./media/portability-analyzer/MissingAssemblies.PNG)
```

Comment:
These screenshots introduce a large problem. They should be in all lowercase. You must do the following to fix this, simply renaming them in Windows will not work with git.

1. Copy the pictures to a temp directory.
2. Delete the pictures here in the repo.
3. Commit the delete changes.
4. Rename the pictures to all lowercase characters
5. Commit the "new" files.

This creates two commits in your git history, one to delete the file and another to recreate it in a new name format.

Then you need to edit this article and make sure the name of the files match the lowercase file name.

Reply:
Fixed